sstables/writer.hh

```diff
@@ -65,16 +78,19 @@ public:
         _out.flush().get();
     }
     // Must be called in a seastar thread.
-    void close() {
-        _out.close().get();
-    }
+    void close();
+
     uint64_t offset() const {
         return _offset.offset;
     }
 
     const writer_offset_tracker& offset_tracker() const {
         return _offset;
     }
+
+    sstring get_filename() const {
+        return _filename.value_or("output_stream");
```

Comment:
I would change _filename to be a plain std::string and in the constructor that doesn't take a filename initialize _filename to "output_stream".

Reply:
sounds good